spid_cie_oidc/onboarding/validators/token_requests.py

```diff
@@ -22,3 +22,20 @@ class TokenAuthnCodeRequest(TokenRequest):
 class TokenRefreshRequest(TokenRequest):
     grant_type: Literal["refresh_token"]
     refresh_token: str
+
+
+class JwtClientAssertionStructure(BaseModel):
+    iss: HttpUrl
+    sub: HttpUrl
+    aud: HttpUrl
+    iat: int
+    exp: int
+    jti: str
+
+
+class JwtClientAssertionStructureSpid(JwtClientAssertionStructure):
+    aud: HttpUrl
```

Comment:
same here as the previous comment

Reply:
in guideline of Spid it seems to be an url no an array

Reply:
yes, we must support both